robocop/run.py

```diff
@@ -248,22 +248,46 @@ def any_rule_enabled(self, checker) -> bool:
 
     def configure_checkers_or_reports(self):
         for config in self.config.configure:
-            if config.count(":") < 2:
+            if config.count(":") < 1:
                 raise robocop.exceptions.ConfigGeneralError(
                     f"Provided invalid config: '{config}' (general pattern: <rule>:<param>:<value>)"
                 )
-            rule_or_report, param, value = config.split(":", maxsplit=2)
-            if rule_or_report in self.rules:
-                rule = self.rules[rule_or_report]
-                rule.configure(param, value)
-            elif rule_or_report in self.reports:
-                self.reports[rule_or_report].configure(param, value)
+            rule_or_report = config.split(":", maxsplit=1)
+            if rule_or_report[0] in self.rules:
+                self._configure_checkers(config)
+            elif rule_or_report[0] in self.reports:
+                self._configure_reports(config)
             else:
-                similar = RecommendationFinder().find_similar(rule_or_report, self.rules)
+                similar = RecommendationFinder().find_similar(rule_or_report[0], self.rules)
                 raise robocop.exceptions.ConfigGeneralError(
-                    f"Provided rule or report '{rule_or_report}' does not exist. {similar}"
+                    f"Provided rule or report '{rule_or_report[0]}' does not exist. {similar}"
                 )
 
+    def _configure_checkers(self, config):
+        if config.count(":") < 2:
+            raise robocop.exceptions.ConfigGeneralError(
+                f"Provided invalid config: '{config}' (general pattern: <rule>:<param>:<value>)"
+            )
+        rule_name, param, value = config.split(":", maxsplit=2)
+        if rule_name in self.rules:
+            self.rules[rule_name].configure(param, value)
+
+    def _configure_reports(self, config):
+        if config.count(":") < 1:
+            raise robocop.exceptions.ConfigGeneralError(
+                f"Provided invalid config: '{config}' (general pattern: <report>:<param>:<value>)"
+            )
+        report_name, param_and_value = config.split(":", maxsplit=1)
+        if report_name == "return_status":
+            if param_and_value.count(":") < 1:
+                raise robocop.exceptions.ConfigGeneralError(
+                    f"Provided invalid config: '{config}' (general pattern: <report>:<param>:<value>)"
+                )
+            param, value = param_and_value.split(":", maxsplit=1)
+            self.reports[report_name].configure(param, value)
+        elif report_name == "timestamp":
+            self.reports[report_name].configure(None, param_and_value)
```

Comment:
Why the difference between timestamp and other reports? For me it could be handled the same any other reports is handled. But I could be missing something

```
robotidy --configure timestamp:format:<format>
robotidy --configure timestamp:timezone:<format>
```

Reply:
Aaah, I was stuck in the thought that both `timezone` and `format` should be configured together... If we drop that, then configure could be like mentioned. That makes implementation much more straightforward.

Reply:
Yeah, they can be configured separetly - the reports are singletons and they preserve all configs. So:

```
robotidy --configure timestamp:format:<format> --configure timestamp:timezone:<timezone>
```
will configure timestamp with format and timezone.

Reply:
Now configurations are handled separately:

Both of default values, ``timezone`` and ``format`` can be configured by
    ``-c/--configure`` and ``timestamp:timezone:"<timezone name>"`` and/or ``timestamp:format:"<format string>"``::

        robocop --configure timestamp:timezone:"Europe/Paris" --configure timestamp:format:"%Y-%m-%d %H:%M:%S %Z %z"

This yields following timestamp report::

         Reported: 2022-07-10 20:38:10 CEST +0200